docs/development/vsphere-dev.md

```diff
@@ -102,7 +102,7 @@ Use .build/dist/linux/amd64/kops if working on a linux machine, instead of mac.
 **Notes**
 
 1. ```clustername``` should end with **skydns.local**. Example: ```kubernetes.cluster.skydns.local```.
-2. ```zones``` should end with ```a```. Example: ```us-west-2a``` or as the above command sets ```--zones=${AWS_REGION}a```.
+2. For ```zones``` any string will do, for now. It's only getting used for the construction of names of various entities. But it's a mandatory argument.
```

Comment:
From kops (aws and gce) point of view default zone doesn't mean anything. I am hoping that we would be able to come up with some meaningful mapping for zone names. If not, later we can think about making it a default. But yes... for now any string will do.